### ISSUE TITLE
€ character cannot be entered

### ISSUE THREAD

**Issue by Tragen** [Genymobile/scrcpy#2511]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows
 - scrcpy version: 1.18
 - installation method: Windows zip
 - device model: OnePlus 7T
 - Android version: 11

**Describe the bug**
When I'm writing a text, everything works, except the € character.
When I press ALTGR+E or CTRL+ALT+E, I get this error in the console.

On errors, please provide the output of the console (and `adb logcat` if relevant).

```
[server] WARN: Could not inject char u+20ac
```

**Comment by rom1v** [comment]
Yes, unfortunately, Android does not support injecting non-ASCII chars on text injection.

It's the same with `adb shell input text €`.

https://github.com/Genymobile/scrcpy/blob/master/FAQ.md#special-characters-do-not-work
Duplicate of #37.

**Comment by Tragen** [comment]
Are you sure? ASCII doesn't include öäüß and I can enter it.

**Comment by rom1v** [comment]
See https://blog.rom1v.com/2018/03/introducing-scrcpy/#handle-accented-characters for explanation.

**Comment by Tragen** [comment]
Thanks. I hope this can be fixed in the future.

**Comment by l1m3r** [comment]
AFAIKT this isn't fixed at all with #2632 
For Linux yes but not for Windows if you require mirroring mode.
I couldn't find any way to enter €, µ, ² or ³ - not even in `--otg --hid-keyboard` mode.

Tried
1. -otg --hid-keyboard -> no mirroring :-(
2. `--serial=<....>` because of #2632
3. `--prefer-text`		-> doesn't work.
4. `--raw-key-events`	-> doesn't work.

Changing the language of the physical OTG keyboard didn't help either.
With 4. I can't get anything else but the English keyboard layout...

**Comment by rom1v** [comment]
@l1m3r Yes, unfortunately, `€` could not be injected from the Android API. I'm also annoyed by that.

With `scrcpy --hid-keyboard` (`scrcpy -K`), it works, but not on Windows because we can't access USB if adb already uses it (on other platforms this is not a problem).

**Comment by l1m3r** [comment]
Maybe there's a way around it - One can _enter_ € and so on via copy/paste.

1. `[server] WARN: Could not inject char u+20ac`
2. disable clipboard-sync temporarily (if it's used).
3. store clipboard content of device temporarily.
4. set device clipboard to just the character from 1. and paste it.
5. restore 3.
6. re-enable 2.

**Comment by rom1v** [comment]
@l1m3r I tried something similar in the paste #1426 but I abandoned the idea (see the thread).
